src/utils/tap/getTapPath.js

```diff
@@ -0,0 +1,19 @@
+const { getGlobalConfig } = require('KegUtils/globalConfig/getGlobalConfig')
+const { get, isStr } = require('@keg-hub/jsutils')
+
+/**
+ * Gets the path to the linked tap with name `name`
+ * @param {string} name - alias name of tap
+ * @param {Object} config - (optional) global config object
+ * @returns {string} path
+ */
+const getTapPath = (name, config) => {
```

Comment:
This method already exists at => `src/utils/globalConfig/getTapPath`
* It works a little different, but does the exact same thing.
* We should eather update that one, or remove it and update all imports of it, to point to this one

Reply:
Updated: I removed my copy and migrated this one into `cli-utils`